libraries/chain/controller.cpp

```diff
@@ -4068,7 +4068,7 @@ struct controller_impl {
       if (if_active) {
          return calculate_merkle( std::move(digests) );
```

Comment:
Search for `calculate_merkle` and you will find a few places where copies are made so they can be moved into `calculate_merkle`. Those copies are no longer needed.

Reply:
done.

Reply:
done